src/web/app/pages-instructor/instructor-home-page/instructor-home-page.component.spec.ts

```diff
@@ -271,6 +280,26 @@ describe('InstructorHomePageComponent', () => {
             .feedbackSession.feedbackSessionName).toEqual('First Session');
   });
 
+  it('should sort courseTabModels by courseId in ascending order', () => {
+    const CopytActiveCourseTabModels: CourseTabModel[] = activeCourseTabModels;
+    CopytActiveCourseTabModels.push({
+      course: testCourse3,
+      instructorPrivilege: testInstructorPrivilege,
+      sessionsTableRowModels: [],
+      sessionsTableRowModelsSortBy: SortBy.NONE,
+      sessionsTableRowModelsSortOrder: SortOrder.ASC,
+
+      hasPopulated: false,
+      isAjaxSuccess: true,
+      isTabExpanded: true,
+      hasLoadingFailed: false,
+    });
+    component.courseTabModels = CopytActiveCourseTabModels;
+    component.sortCoursesBy(SortBy.COURSE_ID);
```

Comment:
Nit: To follow how the other tests are structured.
```suggestion
    component.sortCoursesBy(SortBy.COURSE_ID);

```